manifests/proxydhcp.pp

```diff
@@ -53,6 +53,8 @@
 
   if $foreman_proxy::dhcp_manage_acls {
 
+    File <| title == $::dhcp::dhcp_dir |> { mode => '0750' }
```

Comment:
I think you can also write this as:
```suggestion
    File[$dhcp::dhcp_dir] { mode => '0750' }
```

The benefit is that it doesn't silently ignore it if the resource can't be found.

Reply:
The disadvantage is, that it doesn't work :(
I took the example from [here](https://serverfault.com/questions/438658/how-to-extend-a-file-definition-from-an-existing-module-in-the-node). I'm getting `Only subclasses can override parameters`